packages/common/index.js

```diff
@@ -7,7 +7,6 @@ exports.cliUtils = require('./cli-utils');
 exports.CloudFormationGateway = require('./CloudFormationGateway');
 exports.CollectionConfigStore = require('./collection-config-store').CollectionConfigStore;
 exports.constructCollectionId = require('./collection-config-store').constructCollectionId;
-exports.http = require('./http');
```

Comment:
Is this intentional?

Reply:
No.  Good catch, I'm a bit spooked by that.

Reply:
7867a5255